dyellow-stellenbosch.md

```diff
@@ -0,0 +1 @@
+
```

Comment:
Where is Stellenbosch?  
What is there to do?
What is your favorite restaurant there?